addons/mail/static/src/utils/test_utils.js

```diff
@@ -643,6 +643,14 @@ async function start(param0 = {}) {
                 loadingBaseDelayDuration,
                 messagingBus,
             },
+            /**
+             * Override to ensure tests run in debug mode to catch all potential
+             * programming errors and provide better message if they happen.
```

Comment:
"if" => "when" _(smoother and less intimidating, while fully keeping same meaning)_

Reply:
doesn't really matter to me but ok